test/fuzzing/retrieve_blocks_fuzz.cpp

```diff
@@ -15,7 +26,12 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, std::size_t size) {
   iroha::network::proto::BlocksRequest request;
   if (protobuf_mutator::libfuzzer::LoadProtoInput(true, data, size, &request)) {
     grpc::ServerContext context;
-    fixture.block_loader_service_->retrieveBlocks(&context, &request, nullptr);
+    fuzzing::MockServerWriter serverWriter;
```

Comment:
Should be static, I guess
Moreover some calls probably should be mocked (particularly the ones that return boolean)

Reply:
I don't think it should be static - we pass a non-const raw point to it and have no idea what will happen with it inside.
I'm not sure what call do you mean - the only one which returns boolean is ```Write``` with 2 args but we are not interested in it.